content/updates/2017-05-22-release-notes.md

```diff
@@ -0,0 +1,48 @@
++++
+date = "2017-05-22"
+title = "Platform Release Notes: May 22, 2017"
++++
+
+Curious what’s new that you might find helpful as a cloud.gov application developer? Here are highlights from our platform updates over the past two weeks.
+<!--more-->
+
+### Researching
```

Comment:
I think I'm going to go with `Help make cloud.gov easier to use` since it's pretty directly for users who have a vested interest in that.